frontend/packages/kubevirt-plugin/src/yaml-templates.ts

```diff
@@ -0,0 +1,37 @@
+import { VirtualMachineModel } from './models';
+
+export const vmYamlTemplate = `
```

Comment:
Check out #1613 - a more robust (but also more complex) alternative is to use a map with tuple key `[model, templateName]`:

```tsx
import { Map as ImmutableMap } from 'immutable';

import { FooBarModel } from './models';

export const yamlTemplates = ImmutableMap()
  .setIn([FooBarModel, 'default'], `
apiVersion: ${FooBarModel.apiVersion}
kind: ${FooBarModel.kind}
metadata:
  name: example
  namespace: default
`);
```

I'm OK with a simple `const vmYamlTemplate` too.

In any case, please remove the leading whitespace (they aren't stripped automatically):

```tsx
export const vmYamlTemplate = `
apiVersion: ${VirtualMachineModel.apiGroup}/${VirtualMachineModel.apiVersion}
kind: VirtualMachine
metadata:
  name: example
..
`;
```

Reply:
done